kotlintest-assertions/src/jvmTest/kotlin/com/sksamuel/kotlintest/properties/CharGenTest.kt

```diff
@@ -0,0 +1,15 @@
+package com.sksamuel.kotlintest.properties
+
+import io.kotlintest.properties.Gen
+import io.kotlintest.properties.char
+import io.kotlintest.shouldBe
+import io.kotlintest.specs.FunSpec
+
+class CharGenTest : FunSpec({
```

Comment:
Perhaps adding an exhaustive test that verifies all possible characters are obtained?